MantidQt/SliceViewer/src/QPeaksTableModel.cpp

```diff
@@ -309,14 +310,15 @@ std::vector<int> QPeaksTableModel::defaultHideCols() {
     if (numToCheck > 20) // arbitrary cutoff
       numToCheck = 20;
     const std::string RECT_DET("RectangularDetector");
+    const auto &detectorInfo = m_peaksWS->detectorInfo();
```

Comment:
Didn't pay too much attention to the code itself. Yes, it seems that this bit of code is actually pointless. I will remove it.